### DISCUSSION TITLE
Why the volume is locked to 1000 ?

### DISCUSSION THREAD

**Discussion by hermitoff** [jagrosh/MusicBot#1107] [Questions] [answered]
Hello, I recently used the bot and I noted that it was impossible to set the volume to more than 1000%.
How to pass this limit?

Thank you in advance and here are some pictures: 
![image](https://user-images.githubusercontent.com/82811865/150987115-70227168-5d00-432b-9b9f-1dfde93853d1.png)
![image](https://user-images.githubusercontent.com/82811865/150987685-3895c0c4-c191-4444-8ee4-7d3119de8329.png)

**Comment by Sanduhr32** [comment, marked as answer]
i think lavaplayer limits the volume to 1000.
